docs/motivating_use_case.md

```diff
@@ -119,7 +119,7 @@ allowed_artifact_id_types = [
 
 Let's take a look at one example team's software supply chain.
 
-![drawing](images/attestation_supply_chain.png)
+![drawing](../images/attestation_supply_chain.png)
```

Comment:
does it make sense to remove this for the time being as it's out of date? It might cause confusion.

Reply:
addressed by issue #157